src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/index.test.tsx

```diff
@@ -18,34 +18,63 @@ jest.mock('./settingsComponents/SwitchEditorCard', () => 'SwitchEditorCard');
 jest.mock('./settingsComponents/TimerCard', () => 'TimerCard');
 jest.mock('./settingsComponents/TypeCard', () => 'TypeCard');
 
+jest.mock('../../../../../data/redux', () => ({
```

Comment:
Can we pass this as `initialState` to `editorRender` instead of mocking the whole redux file? Or was that not working for some reason?